workflow/usage.rst

```diff
@@ -255,8 +255,9 @@ order:
     * ``workflow.[workflow name].transition.[transition name]``
 
 ``workflow.enter``
-    The object entered a new place. This is the first event where the object
-    is marked as being in the new place.
+    The object is about to enter a new place. This is the event where the object is
+    going to be marked as being in the new place.
+    Please notice: the marker store of the object is not yet updated with the new state.
```

Comment:
Nice catch. But the vocabulary is not accurate.
> Please notice : the marking of the object is not ...

The `marking` is the representation of the subject in the workflow
The `marking store` is the interface between the subject and the workflow
The `marker` does not exist

s/state/places/ (plural!)